querybook/server/clients/glue_client.py

```diff
@@ -0,0 +1,150 @@
+from typing import List
+
+import boto3
+from lib.logger import get_logger
+
+_LOG = get_logger(__file__)
+
+
+class GlueDataCatalogClient:
+    def __init__(self, catalog_id, region="us-east-1"):
+        self.catalog_id = catalog_id
+        self._glue_client = boto3.client("glue", region_name=region)
+
+    def __del__(self):
+        del self._glue_client
+
+    def get_all_databases(self):
+        """
+        :return: The objects for all the Glue databases
+        """
+        _LOG.info("Get all databases")
+
+        paginator = self._glue_client.get_paginator("get_databases")
+        result = {}
+        database_list = []
+
+        for page in paginator.paginate(CatalogId=self.catalog_id):
+            database_list.extend(page.get("DatabaseList"))
+
+        result["DatabaseList"] = database_list
+
+        return result
+
+    def get_all_database_names(self):
+        """
+        :return: The names of all the Glue databases
+        """
+        _LOG.info("Extracting all databases names")
+
+        databases = self.get_all_databases()
+        database_list = databases.get("DatabaseList")
+
+        result = list(map(lambda database: database.get("Name"), database_list))
```

Comment:
```
result = [database.get("Name") for database in database_list]
```
I saw in the code below there are uses of both lambda syntax and list comprehension syntax, do you want to stick to list comprehension since it's more compact?

Reply:
Thank you @czgu for having a look.
I will adjust the code later in the evening :)